src/courses/course1/index.js

```diff
@@ -1,27 +1,13 @@
-const uuid = require('uuid');
 const page1 = require('./page1');
 const page2 = require('./page2');
 const page3 = require('./page3');
 const page4 = require('./page4');
 const page5 = require('./page5');
 const page6 = require('./page6');
-
-const pagelist = [page1, page2, page3, page4, page5, page6];
-
-for (let i = 0; i < pagelist.length; i += 1) {
-  if (i === 0) {
-    pagelist[i].next = pagelist[i + 1].id;
-  } else if (i === pagelist.length - 1) {
-    pagelist[i].previous = pagelist[i - 1].id;
-  } else {
-    pagelist[i].next = pagelist[i + 1].id;
-    pagelist[i].previous = pagelist[i - 1].id;
-  }
-}
+const page7Intro = require('./page7intro');
```

Comment:
Naming is a bit weird, a new page 7/8 might be better

Reply:
these are just place holders, once we have a sample course of pages we should really name them along the lines of what the page is about